database/data-generator.js

```diff
@@ -0,0 +1,54 @@
+const db = require('./index.js');
+const _ = require('lodash');
+
+const listingSizeDescription = [
+  'ENTIRE GUEST SUITE',
+  'ENTIRE APARTMENT',
+  'ENTIRE HOUSE',
+  'ENTIRE BUNGALOW',
+  'ENTIRE TOWNHOUSE',
+  'PRIVATE ROOM',
+  'ENTIRE LOFT',
+];
+const listingHeader = [
+  'Malibu boutique apartment',
+  'Beautiful OCEAN FRONT BEACH condo!',
+  'Tree house with 360 views of L.A',
+  'OVER THE WATER Black - Contemporary Bungalow',
+  'Stunning Villa by the Sea w/ Direct Beach Access!',
+  'Modern Oceanview w/ Big Patio',
+  'Rustic, Romantic Beachfront Apartment in Malibu',
+  'Oceanv View Malibu Hideaway',
+  'Ocean Front Malibu House',
+  'Malibu Ocean View Rooms',
+  'Beautiful Malibu Beach Duplex',
+  'Ocean View Malibu Mountain Retreat',
+];
+const ratings = _.range(1, 5.5, 0.5);
+
+for (let i = 1001; i <= 1100; i += 1) {
```

Comment:
Generated yourself! Impressive! Curious what was the decision to not use mockaroo or faker?

Reply:
Thank you! For my particular use case, It just seemed that it would take longer to read the docs on how to use another module/service than to create the mock data myself.